pkgs/development/libraries/gdal/default.nix

```
@@ -20,22 +20,22 @@ stdenv.mkDerivation rec {
 
   sourceRoot = "source/gdal";
 
-  nativeBuildInputs = [ autoreconfHook ];
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
 
   buildInputs = [ unzip libjpeg libtiff libpng proj openssl sqlite
-    libspatialite libgeotiff poppler hdf4 qhull giflib expat libxml2 ]
-  ++ (with pythonPackages; [ python numpy wrapPython ])
-  ++ stdenv.lib.optional stdenv.isDarwin libiconv
-  ++ stdenv.lib.optionals netcdfSupport [ netcdf hdf5 curl ];
+    libspatialite libgeotiff poppler hdf4 qhull giflib expat libxml2 postgresql]
+    ++ (with python3Packages; [ python numpy wrapPython ])
+    ++ stdenv.lib.optional stdenv.isDarwin libiconv
+    ++ stdenv.lib.optionals netcdfSupport [ netcdf hdf5 curl ];
 
-  configureFlags = [
+ configureFlags = [
```

```suggestion
  configureFlags = [
```

Preferably you would not change the formatting, or change it in the whole file using something like a nixpkgs-fmt in a separate commit.